WhatsOnTap/Establishment.h

```diff
@@ -0,0 +1,16 @@
+//
+//  Establishment.h
+//  WhatsOnTap
+//
+//  Created by Sarmila on 6/14/16.
+//  Copyright © 2016 Detroit Labs. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface Establishment : NSObject
+@property (strong, nonatomic) NSString *EstablishmentName;
```

Comment:
next time someone works in this file, that needs to be changed to establishmentName